pkg/library/overrides/testdata/container-overrides/overrides-can-override-securityContext.yaml

```diff
@@ -0,0 +1,26 @@
+name: "container overrides can override securityContext"
```

Comment:
This test name as well as `/container-overrides/overrides-can-use-delete-directive.yaml` and `/container-overrides/overrides-can-use-replace-directive.yaml` starts with a lower-case letter rather than uppercase like the others.

Very unimportant and doesn't need to be fixed, but it's the only remark I could find about this PR :P